### ISSUE TITLE
make install installs Google test libs but not NativeJIT

### ISSUE THREAD

**Issue by joto** [BitFunnel/NativeJIT#63]
`make install` installs `gtest` and `gmock` libs and includes, which should probably not be installed, but it doesn't install `NativeJIT` and `CodeGen` libs and includes.

**Comment by danluu** [comment]
Good catch! We obviously don't include NativeJIT this way so the it just happens to do this by coincidence since we weren't thinking about this when we put together the CMakeLists.txt.

If no one else knows the right way to do this, we'll take a look at fixing up our CMakeLists to do the right thing. Sorry about this! We're relatively new to using cmake.
